RockWeb/Webhooks/Mailgun.ashx

```diff
@@ -78,19 +78,26 @@ public class Mailgun : IHttpHandler
             
             if ( !string.IsNullOrWhiteSpace( request.Form["workflow_action_guid"]))
             {
-                Guid? actionGuid = request.Form["workflow_action_guid"].AsGuidOrNull();
+                Guid? actionGuid = request.Form["workflow_action_guid"].Split( ',' )[0].AsGuidOrNull();
                 string status = string.Empty;
                 switch ( eventType )
                 {
-                    case "complained":
-                    case "unsubscribed":
+                    case "complained": break;
+                    case "unsubscribed": break;
                     case "delivered": status = SendEmailWithEvents.SENT_STATUS; break;
```

Comment:
Adding the breaks, would cause the status not to get set to 'SENT' when a complained or unsubscribed event was relieved. Without the 'break' these fall through to the "delivered" case.